scanner/src/main/kotlin/ScanController.kt

```diff
@@ -169,7 +169,7 @@ class ScanController(
     /**
      * Get the [NestedProvenanceScanResult] for the provided [id].
      */
-    fun getNestedScanResult(id: Identifier): NestedProvenanceScanResult =
+    fun getNestedScanResult(id: Identifier): NestedProvenanceScanResult? =
```

Comment:
Commit message: You forgot to mention which function you refer to in the first sentence (`buildNestedProvenanceScanResult()`).